trade-finance-manager-api/src/v1/rest-mappings/filters/filterTasks.api-test.js

```diff
@@ -200,6 +208,29 @@ describe('filterTasks', () => {
   });
 
   describe('filterTasks function', () => {
+    describe('when there are no tasks', () => {
+      it('should return an empty array back when there are no tasks', () => {
+        const mockFiltersObj = {};
+
+        const result = filterTasks(mockEmptyTasks, mockFiltersObj);
+        expect(result).toEqual([]);
+      });
+
+      it('should return an empty array back when the task is null', () => {
+        const mockFiltersObj = {};
+
+        const result = filterTasks(null, mockFiltersObj);
+        expect(result).toEqual([]);
+      });
+
+      it('should return an empty array back when tasks are undefined', () => {
```

Comment:
Same in these tests @abhi-markan 

```suggestion
      it('should return an empty array when tasks are undefined', () => {
```